src/components/ReactContextMenu.js

```diff
@@ -24,10 +19,11 @@ export default class ContextMenu extends React.Component {
             const clickHandler = () => {
               this.closeContextMenu();
               item.function(this.state.target);
-            }
+            };
 
             const label = item.label;
             const icon = item.icon;
+
```

Comment:
To już wyłącznie moje zboczenie, ale dałoby się to zdestrukturyzować:
`const { label, icon } = item;`